### PR TITLE
Add company values and platform pricipals

### PR DESCRIPTION
**Pull request by codingjoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,81 @@ This document contains all information related to our software engineering,
...
+
+#### 5. Unite people, don't divide.
+
+In a world that is more and more divided, we want to bring people together.
+We don't judge people, we educate them. We don't blame people, we help them.
+
+#### 6. Give strength, when others need it.
+
+We take care of our own. We are here for you, should you need our help,
+but we expect you give strength to others, whenever you can.
```
Comment (herrbenesch) [inline README.md:59]:
```suggestion
We take care of our own. We are here for you, should you need our help.
But we expect you give strength to others, whenever you can.
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,81 @@ This document contains all information related to our software engineering,
...
+#### 1. thoughtfulness
+
+We care about the details. Every interaction should be thoughtful and intentional.
+A user should feel that we deeply care about the product we are building.
+Small things matter.
+
+#### 2. inspiring
+
+We want to inspire people to do new things. We build products and services
+that exceed our customers wildest dreams. Cars not faster horses.
```
Comment (herrbenesch) [inline README.md:74]:
```suggestion
that exceed our customers wildest dreams. Cars, not faster horses.
```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,81 @@ This document contains all information related to our software engineering,
...
+#### 6. Give strength, when others need it.
+
+We take care of our own. We are here for you, should you need our help,
+but we expect you give strength to others, whenever you can.
+
+### Our Product Principles
+
+Our product principles are the secret sauce that makes our products extra special.
+
+#### 1. thoughtfulness
```
Comment (herrbenesch) [inline README.md:65]:
```suggestion
#### 1. thoughtful
```

**Review by herrbenesch — APPROVED** [review]
Nice. I added some improvement proposals.

File: README.md [hunk 4]
```diff
@@ -8,6 +8,81 @@ This document contains all information related to our software engineering,
...
+#### 3. There are no silver medals.
+
+We get up in the morning to run for gold, not for silver.
+We will fail, but we don't start something with a failure mindset.
+
+#### 4. If you gotta work, it better be fun!
+
+Work is a major part of our lives. Therefore, we want to do work,
+that fulfills us and we enjoy. If we don't, we are but a penguin on land –
+not in our element.
```
[the remark this answers is not in the document]
Comment (codingjoe) [reply README.md:49]: They do, but they look an act very clumsy, where they are majestic flyers underwater.

**Pull request merged**